app/database.py

```diff
@@ -0,0 +1,107 @@
+from os import remove
+from os.path import isfile
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from termcolor import cprint
+
+from .room import LivingSpace, Office
+from .person import Fellow, Staff
+from . import dojo
+from .tables import RoomDB, PersonDB, Base
+
+class Database:
+
+	def __init__(self):
+		self.dojo = dojo
+
+	def save_state(self, db_name="dojo.db"):
+		try:
+			remove(db_name)
+		except Exception:
+			pass
+		engine = create_engine("sqlite:///" + db_name, echo = False)
+		Base.metadata.bind = engine
+		Base.metadata.create_all(engine)
+		DBSession = sessionmaker(bind=engine)
+		session = DBSession()
+
+		for room in self.dojo.rooms:
+			session.add(RoomDB(name = room.name, category = room.category,
```

Comment:
`session.add(RoomDB(
				name = room.name,
				category = room.category,
				available_capacity = room.available_capacity)
			)`
this indentation would look neater. Check pep8